README.md

```diff
@@ -84,6 +84,8 @@ Developer documentation is generated using the JSDoc library. To generate a loca
 
 ## Migrating Database Changes
 
+> :warning: **WARNING:** Releases which include database migrations should also include an update of the demo db dump. See [instructions](./demo/README.md) on how to create and commit the new dump.
```

Comment:
I'm not sure this is needed.
We should be running the migrations against the demo db right before deploying.
A "job" should be ran after the db container has started and before the IPR API container has started that runs the migrations and then exits.

Reply:
This warning is for the current procedure. Once/If the procedures changes or is removed then the documentation can be updated at the same time.